package.json

```diff
@@ -27,19 +27,23 @@
   "homepage": "https://github.com/danielbayerlein/dashboard#readme",
   "dependencies": {
     "babel-plugin-styled-components": "^1.1.7",
+    "cross-env": "^5.0.5",
     "dotenv-webpack": "^1.5.4",
     "isomorphic-unfetch": "^2.0.0",
     "js-base64": "^2.1.9",
+    "module-alias": "^2.0.1",
     "next": "^3.0.6",
     "polished": "^1.6.1",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1",
```

Comment:
Is `react` and `react-dom` not required? See https://github.com/zeit/next.js/pull/2583

Reply:
in production only preact and preact-compat is required, there for react and react-dom will only be shipped via yarn/npm in dev :)

if you clean your node_modules and  reinstall it with yarn install --prod, then it also works with yarn build and yarn start :)

Reply:
Merge error or do we need both dependencies?

```
"preact": "^8.2.5",
"preact-compat": "^3.17.0",
"react": "^15.6.1",
"react-dom": "^15.6.1"
```

Reply:
merge error - i fixed it :)